grpc-testtool/proto/merkle/merkle.proto

```diff
@@ -0,0 +1,102 @@
+syntax = "proto3";
+
+package merkle;
+
+import "google/protobuf/empty.proto";
+
+service Merkle {
+    rpc NewProposal(NewProposalRequest) returns (NewProposalResponse);
+    rpc ProposalCommit(ProposalCommitRequest) returns (ProposalCommitResponse);
+  
+    rpc NewView(NewViewRequest) returns (NewViewResponse);
+    // The methods below may be called with a view ID that corresponds to either a (committable) proposal
+    // or (non-committable) historical view.
+    rpc ViewHas(ViewHasRequest) returns (ViewHasResponse);
+    rpc ViewGet(ViewGetRequest) returns (ViewGetResponse);
+    rpc ViewNewIteratorWithStartAndPrefix(ViewNewIteratorWithStartAndPrefixRequest) returns (ViewNewIteratorWithStartAndPrefixResponse);
+    rpc ViewRelease(ViewReleaseRequest) returns (google.protobuf.Empty);
+}
+
+message NewProposalRequest {
+  // If not given, the parent view is the current database revision.
+  optional uint32 parent_view_id = 1;
+  repeated PutRequest puts = 2;
+  repeated bytes deletes = 3;
```

Comment:
Should be a DeleteRequest, also from rpcdb.proto

Reply:
I'd be fine having this just be `repeated bytes` instead of `repeated DeleteRequest` where `DeleteRequest` just has one `bytes` field. If you feel strongly I can add that type though

Reply:
Ah, I see, that's just the key to delete. It's fine.

nit: comment that this is the key value